src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -430,6 +438,9 @@ public void removeDataBundle(DataBundle dataBundle) {
             }
         }
         accountsDb.deleteAccounts(dataBundle.accounts.values());
+
+        // Delete all admin emails
+        adminEmailsDb.deleteEntities(adminEmailsDb.getAllAdminEmails());
```

Comment:
@leeyimin I just realized what this line entails. If run on an actual server, this will effectively wipe out all admin emails?

I may be wrong, but perhaps you're thinking of:
```java
adminEmailsDb.deleteEntities(dataBundle.adminEmails.values());
```